plugins/modules/github_org_repository.py

```diff
@@ -430,7 +431,7 @@ class GHOrgRepositoryModule(GitHubBase):
                     options=dict(
                         users=dict(type='list', elements='str', default=[]),
                         teams=dict(type='list', elements='str', default=[]),
-                        apps=dict(type='list', elements='str')
+                        apps=dict(type='list', elements='str', default=[])
```

Comment:
This change should not be necessary since GH does not require this options to be set. We have defaults set where GH requires this option to be set

Reply:
I can't remember exactly why I added it, but it must have been for some reason (it was quite a long time ago). From my point of view, at least, it shouldn't have done any harm. I would therefore recommend leaving it as it is.

Reply:
Removed.